deepfence_server/router/router.go

```diff
@@ -258,6 +258,9 @@ func SetupRoutes(r *chi.Mux, serverPort string, serveOpenapiDocs bool, ingestC c
 				r.Post("/host", dfHandler.CompleteHostInfo)
 			})
 
+			r.Route("/filter-hosts", func(r chi.Router) {
```

Comment:
All filter completion are defined here:
https://github.com/deepfence/ThreatStrykerV2/blob/3ea37f06dbece6d4760f4796444a0d2d50af734d/deepfence_server/router/router.go#L276-L280

Please move yours inside the same namespace.

I see we already provide hosts completion, we should reuse that (add a request field for dead entries) instead of reinventing the same method

Reply:
alright!